src/test/java/testresults/expiry_date/PostTestResultsFirstExpiryDatesHgv_12215.java

```diff
@@ -58,10 +58,9 @@ public void setTestCode(String testCode) {
     }
 
 
-    @Ignore
     @WithTag("expiry_dates")
     @Title("CVSB-12215 - AS a VSA I want the first expiry date to be calculated using reg/ first use anniversary so that it is accurately recorded for annual tests - HGV - NO Previous Expiry Date - regnDate = null")
-    @Test
+
```

Comment:
If you remove the @Test tag here we should put back the @Ignore tag so that is clear that the test is ignored from execution. The removal of the @Test tag is done just for the test to not appear in the test report.